WalletWasabi.Tests/UnitTests/BranchAndBoundSelectionTests.cs

```diff
@@ -0,0 +1,51 @@
+using NBitcoin;
+using System.Collections.Generic;
+using WalletWasabi.BranchNBound;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
+{
+	public class BranchAndBoundSelectionTests
+	{
+		private Random Random { get; } = new();
+
+		[Fact]
+		public void RandomizedTest()
+		{
+			var utxos = GenList();
+			var selector = new BranchAndBound(utxos);
```

Comment:
In new code, I would recommend https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/target-typed-new notation, so it would be:

```suggestion
			BranchAndBound selector = new(utxos);
```

And the reason why I find it useful is because:

```cs
			var selector = new BranchAndBound(utxos);
```

is equivalent to

```cs
			BranchAndBound? selector = new BranchAndBound(utxos);
```

and that is most likely something you do not want to write really.

Reply:
You are absolutely right. It was in my mind, but didn't have time fix it yet.